app/src/main/java/corp/kairos/adamastor/Settings/ContextRelated/ContextRelatedAppsActivity.java

```diff
@@ -50,7 +45,6 @@ private void loadSettings(Bundle savedInstanceState) {
         } else {
             this.ctx = (String) savedInstanceState.getSerializable("CONTEXT");
         }
-        this.settingsUser = Settings.getInstance(this);
```

Comment:
you cannot remove this line, if you do so, the app crashes when you try to select the context related apps, because userSettings then is null and will be an error bellow